### PR TITLE
[WIP] # 393 fixing Added language

### PR DESCRIPTION
**Pull request by STEdirisuriya**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

### Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
  
  http://rawgit.com/STEdirisuriya/labyrinth/master/index.html
- 
https://github.com/fossasia/labyrinth/issues/393

### The problem I want to solve or the facility I want to improve is/are
Added way to translate game and added sinhala translate 

### My steps to solve it
Use js file

### CODE REVIEW

**Comment by STEdirisuriya** [comment]
# don't merge this yet. I'm working yet.

**Comment by STEdirisuriya** [comment]
Index.heml file translating finished
how to play, contributors, credits, tiles, .. more not finished

**Comment by STEdirisuriya** [comment]
contributors page translated

File: js/translate.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+$(function () {
+  $('.translate').click(function () {
+    var lang = $(this).attr('id');
+
+    $('.lang').each(function (index, element) {
+      $(this).text(arrlang[lang][$(this).attr('key')]);
+    });
+  });
+});
```
Comment (niccokunzmann) [inline js/translate.js]: cool, yes, this should work.

**Comment by niccokunzmann** [comment]
@STEdirisuriya This looks totally good to me. Time will tell if we need other features but I think this is fine.

**Comment by STEdirisuriya** [comment]
I finished credits page translating. But I have a problem , If it's about credits there must be credits of all the maintainers and contributors. but there is a license file, It's should be removed, That's why I not translated it, and also it's not suitable for license. see more http://rawgit.com/fossasia/labyrinth/master/credits.html
@niccokunzmann

**Comment by STEdirisuriya** [comment]
only I have to translate ho to play

**Comment by STEdirisuriya** [comment]
30-40 mins only

**Comment by niccokunzmann** [comment]
You do not have to do that. You can create an issue and use this to spead the knowledge about how to translate. This is healthy for the project and allows other people to better use your code.

**Comment by STEdirisuriya** [comment]
pls give me only 10-20mins

**Comment by STEdirisuriya** [comment]
almost done

**Comment by niccokunzmann** [comment]
You have all the time... and merge conflicts.

**Comment by STEdirisuriya** [comment]
finished as I think @niccokunzmann

**Comment by niccokunzmann** [comment]
Looks like translations work.

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**